css/style.css

```diff
@@ -246,6 +252,22 @@ header #burger-button:hover + nav ul li a {
   background-color: white;
   border-bottom-left-radius: 100px;
   background-image: url("../img/header-shapes\ mobile.jpg");
+  background-size: cover;
+}
+
+@media (min-width: 786px) {
+  #landing-page {
+    background-image: url("../img/Header\ bg.svg");
+  }
+}
```

Comment:
- [ ] In the Header I recommend adjusting the alignmenet for the heading to match the figma template and the paragrph because it is very close to the circles that on the background so kindly fix it I will show you an image 🤗💯💪🏻 

- Your design
![image](https://user-images.githubusercontent.com/58553711/193037846-7e72363d-6be0-4757-98fb-c001cb5a8206.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/193037988-47999347-179d-4cc5-8563-8ac2c33580c9.png)